### PR TITLE
refactor(LoginCard): redesign page

### PR DESCRIPTION
**Pull request by AGDholo**
## Description of the changes

- Move login to auth/login
- Add icons
- Add dark support in LangPicker
- Add global scss(style/app.scss)
- Redesign login page

Same my project design from: https://vadmin.heroui.net/auth/login

image

image

image

image

image

image

### CODE REVIEW

**Comment by izgzhen** [comment]
The design looks great! I haven't looked at code yet. Can you also fix the CI failure and request my review after the code is ready? Thanks!

**Comment by AGDholo** [comment]
> Can you also fix the CI failure.

The CI check failed is not caused by this PR, but I still try to fix it, and more testing may be needed, because I don’t know whether the repaired lint is available.

**Comment by izgzhen** [comment]
The log shows that it is prettier who is not happy (it is in incremental mode so ....) you can fix it with

```
yarn run prettier -w .
```

**Comment by izgzhen** [comment]
I just updated your branch to keep to date with `master` in Github UI as well.

File: src/common.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
-export const URLRegex = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}(\.[a-zA-Z0-9()]{1,6})?\b([-a-zA-Z0-9()@:,%_\+.~#?&//=]*)/gi;
```
Comment (izgzhen) [inline src/common.ts:1]: Why?

File: src/api/article.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const apiArticle = {
       authHeaders(token)
     );
   },
-  async getArticleArchives(token: string, articleUUID: string, offset: number, limit: number) {
```
Comment (izgzhen) [inline src/api/article.ts:63]: nit: I was intending to implement it soon so left a few unused parameters. ESlint might complain about it but we can ignore warnings for now (it is not blocking the CI).

File: src/common.ts (hunk 1 above)
Comment (izgzhen) [reply src/common.ts:1]: I remember that ESint might have some warning about it -- however I am not sure whether the warning makes sense. We need to escape `+` in regex right?

File: src/main.ts [hunk 3]
```diff
@@ -1015,14 +1027,12 @@ const i18n = new VueI18n({
       'Other Circles': '其他圈子',
       'New reply': '新回复',
       'Write reply': '回复评论',
+      "Haven't account": '还没账户',
```
Comment (izgzhen) [inline src/main.ts:1030]: `No account`?

**Review by izgzhen — CHANGES_REQUESTED** [review]
Some minor issues. I will test the code later as well.

File: src/main.ts (hunk 3 above)
Comment (AGDholo) [reply src/main.ts:1030]: yeah

File: src/common.ts (hunk 1 above)
Comment (AGDholo) [reply src/common.ts:1]: IDE auto fix, I don't know

**Comment by izgzhen** [comment]
It’s probably a wrong auto fix then

**Comment by AGDholo** [comment]
Should I rollback all code or fix some review block?

**Comment by izgzhen** [comment]
> Should I rollback all code or fix some review block?

I typically will just add a new commit addressing the issues. No need to roll back. Commit history is not so important right now.

**Comment by AGDholo** [comment]
> The log shows that it is prettier who is not happy (it is in incremental mode so ....) you can fix it with
> 
> ```
> yarn run prettier -w .
> ```

No useful, yarn check:eslint have some problems too

**Comment by izgzhen** [comment]
> > The log shows that it is prettier who is not happy (it is in incremental mode so ....) you can fix it with
> > ```
> > yarn run prettier -w .
> > ```
> 
> No useful, yarn `check:eslint` have some problems too

I see. Then just let me add some commit later to this branch to address the CI issue.

File: src/api/article.ts (hunk 2 above)
Comment (AGDholo) [reply src/api/article.ts]: Rollback this line

File: src/common.ts (hunk 1 above)
Comment (AGDholo) [reply src/common.ts]: Rollback this line

**Review by izgzhen — APPROVED** [review]

**Comment by izgzhen** [comment]
Thanks!

**Pull request merged**